src/Employer.Web/Configuration/IoC.cs

```diff
@@ -16,24 +18,45 @@ public static void AddIoC(this IServiceCollection services, IConfiguration confi
             //Configuration
             services.Configure<ExternalLinksConfiguration>(configuration.GetSection("ExternalLinks"));
             services.Configure<AuthenticationConfiguration>(configuration.GetSection("Authentication"));
+            services.Configure<AccountApiConfiguration>(configuration.GetSection("AccountApiConfiguration"));
+            
+            RegisterAccountApiClientDeps(services);
 
             services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>(); // Used by NLog to log out traceidentifier value.
 
-            services.AddTransient<IGetAssociatedEmployerAccountsService, GetAssociatedEmployerAccountsService>();
+            if (configuration.GetValue<bool>("UseStubs"))
```

Comment:
Do we need to add this configuration option?  For the Mongo repo I've just checked to see if I have Mongo configuration and if I haven't then I just use the stub.  Can/should we do something similar here?

Reply:
I was trying it but felt it wasn't explicit enough for me. This `UseStubs` approach isn't something I am enforcing but found while working disconnected was going to suit me better for the moment.